src/masoniteorm/config.py

```diff
@@ -11,11 +11,16 @@ def load_config(config_path=None):
         1. try to load from DB_CONFIG_PATH environment variable
         2. else try to load from default config_path: config/database
     """
+    env_path = os.getenv("DB_CONFIG_PATH", None)
 
-    if not os.getenv("DB_CONFIG_PATH", None):
-        os.environ["DB_CONFIG_PATH"] = config_path or "config/database"
+    if env_path:
+        selected_config_path = env_path
+    elif os.getenv("TESTING", "").lower() == "true":
```

Comment:
I think we need to name this something a little more obscure. This is only for developing the library right? If someone in their application has a environment variable calling TESTING (very possible) it will break if they don't have a config/test-database.py file. Can we name this something like `ORM_TESTING`?